### DISCUSSION TITLE
连接到Websocket服务器时出现错误

### DISCUSSION THREAD

**Discussion by BuShouFan** [FloatTech/ZeroBot-Plugin#907] [Q&A] [answered]
![image](https://github.com/FloatTech/ZeroBot-Plugin/assets/126171728/e9db82a3-51d5-41af-a55a-79a1a59792db)
我想要使用LLOneBot，但是ZBP提示连接到Websocket服务器出现错误，请问我该怎么解决，我看到Discussions里已经有人做了类似问题的回答，但是我没看懂

**Comment by YumeMichi** [comment, marked as answer]
你LLOneBot要用正向WS

**Reply by BuShouFan** [reply]
感谢，我LLOneBot忘记应用更改了
